ftdetect/puppet.vim

```diff
@@ -11,5 +11,14 @@ au! BufRead,BufNewFile *.pp setfiletype puppet
 " Some epp files may get marked as "mason" type before this script is reached.
 " Vim's own scripts.vim forces the type if it detects a `<%` at the start of
 " the file. All files ending in .epp should be epuppet
-au BufRead,BufNewFile *.epp setl ft=epuppet
+if !has('nvim-0.8.0')
```

Comment:
now that there's a .lua file, does this need to specifically exclude nvim 0.8.0+ ? e.g. will nvim versions prior to 0.8.0 not be able to use the lua script? (asking out of total ignorance about this)

Reply:
I don't know, I can't remember if I've tested older nvim without this 'if'.